.github/workflows/optimize-images.yml

```diff
@@ -0,0 +1,61 @@
+# Release builds that are triggered by the v*.*.* tag. Release tags occur when
+# someone comments /approve on the associated release ticket
+
+name: '[Build] Release Build'
+
+on:
+  push:
+    branches:
+      - 'release/v[0-9]+.[0-9]+.[0-9]+'
+      - 'chanel-6809-add--mage-optimization-into-ci-for-app-store-submissions'
+
+jobs:
+
+  optimize_images:
+    name: Optimize Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install imagemin-cli
+        run: npm install -g imagemin-cli
+      - name: List images before optimization
+        run: |
+          echo "iOS images before optimization:"
+          ls -lh VAMobile/ios/fastlane/screenshots/en-US
+          echo "Android images before optimization:"
+          ls -lh VAMobile/android/fastlane/metadata/android/en-US/images/
+          ls -lh VAMobile/android/fastlane/metadata/android/en-US/images/phoneScreenshots/
+          ls -lh VAMobile/android/fastlane/metadata/android/en-US/images/sevenInchScreenshots/
+      - name: Optimize iOS images
+        run:  imagemin VAMobile/ios/fastlane/screenshots/* --out-dir=VAMobile/ios/fastlane/screenshots/en-US
+      - name: Optimize Android images
+        run: |
+          imagemin VAMobile/android/fastlane/metadata/android/en-US/images/*.{png,jpg,jpeg,gif,svg} --out-dir=VAMobile/android/fastlane/metadata/android/en-US/images/
+          imagemin VAMobile/android/fastlane/metadata/android/en-US/images/phoneScreenshots/*.{png,jpg,jpeg,gif,svg} --out-dir=VAMobile/android/fastlane/metadata/android/en-US/images/phoneScreenshots/
+          imagemin VAMobile/android/fastlane/metadata/android/en-US/images/sevenInchScreenshots/*.{png,jpg,jpeg,gif,svg} --out-dir=VAMobile/android/fastlane/metadata/android/en-US/images/sevenInchScreenshots/
```

Comment:
Consider a `cd VAMobile/android/fastlane/metadata/android/en-US/images` before running imagemin so that these lines are shorter and easier to read.